GDSCommon-Demo/GDSCommon-Demo/Mocks/Screens.swift

```diff
@@ -63,8 +64,7 @@ enum Screens: String, CaseIterable {
                                                                screenView: {}, dismissAction: {})
             return InstructionsWithImageViewController(viewModel: viewModel)
         case .gdsModalInfoView:
-            let viewModel = MockModalInfoViewModel()
-            let view = ModalInfoViewController(viewModel: viewModel)
+            let view = ModalInfoViewController(viewModel: MockModalInfoViewModel())
```

Comment:
Refactored here and line 95, as swift lint was complaining that the function was over 50 lines.